integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java

```diff
@@ -850,6 +850,92 @@ private void createPV() throws Exception {
     new PersistentVolume("/scratch/acceptance_test_pv/persistentVolume-" + domainUid, pvMap);
   }
 
```

Comment:
Need description of the method in javadoc style.

Reply:
Done